resolwe_bio/processes/workflows/rnaseq.yml

```diff
@@ -229,3 +229,76 @@
           mode: union
           stranded: '{{input.stranded}}'
           id_attribute: '{{input.id_attribute}}'
+
+- slug: dss-rnaseq
```

Comment:
In previous process the slug is `rna-seq`, here is `dss-rnaseq`. Write RNA-seq constantly with dash or without. So either use `rnaseq` and `dss-rnaseq` or `rna-seq` and `dss-rna-seq`. @jkokosar Which do you prefer?